code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -321,6 +322,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/newscaster, 30)
 					break
 			current_channel.toggleCensorDclass()
 
+		if("printNewspaper")
+			if(paper_remaining <= 0)
+				balloon_alert_to_viewers("Unit out of Paper!")
```

Comment:
This should be a balloon alert just to the user, since only they are trying to print to the paper.

That, or a `say`. A lot of these seem to be `say`. That's fine.

If you do want a balloon alert, change to "out of paper!"